g10k.go

```diff
@@ -244,7 +246,7 @@ func main() {
 	version := *versionFlag
 
 	if version {
-		fmt.Println("g10k version 0.8.9 Build time:", buildtime, "UTC")
+		fmt.Println("g10k version 0.8.10 Build time:", buildtime, "UTC")
```

Comment:
This shouldn't be part of the PR. The g10k author might want to bundle more changes before releasing a new version.